src/github/client/github-queries.ts

```diff
@@ -9,6 +9,42 @@ query {
 	}
 }`
 
+type RepositoryNode = {
+  node: Repository
+}
+
+export type GetRepositoriesResponse = {
+  viewer: {
+    repositories: {
+      pageInfo,
+      edges: RepositoryNode[]
+    }
+  }
+};
```

Comment:
We could simply use types from Octokit for responses like we do in other queries.

But this one even better 👍